internal-api/src/main/proto/com/makarytskyi/internalapi/commonmodels/repairing/repairing.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package com.makarytskyi.internalapi.commonmodels.repairing;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+
+message Repairing {
+  enum RepairingStatus {
```

Comment:
Pls add [prefix](https://protobuf.dev/programming-guides/proto3/#enum-default) to all of your statuses